### ISSUE TITLE
Clone Creation Error

### ISSUE THREAD

**Issue by fabioloretti** [eirannejad/pyRevit#1029]
Hi Eiran,
I've been through all the documentation about creating clones, configuration, deployments, etc etc, but I'm honestly struggling with the clone creation and I was hoping you could shed some light.
I downloaded a master ZIP which I'd like to use to deploy clones in several users' computers but when I test it on my personal laptop after installing the CLI I get an "Error: Can not find clone 'planner'". Below is an extract of the PS1 file I'm running.
Any help is very much appreciated. Thank you.

# set the global variables
$cloneName = 'planner'
$pyRevitSource = 'someServerLocation\pyRevit-master.zip'
$pyRevitRoot = 'C:\pyRevit'
$pyRevitExts = 'someServerLocation\Extensions'

# clone pyRevit
pyrevit clone $cloneName core --source=$pyRevitSource --dest=$pyRevitRoot --log=$logFile

**Comment by fabioloretti** [comment]
Update: if I remove the source path I am able to create the clone directly from github but I'd rather avoid it as it involves downloading about 150MB of data for each user and I'd much rather do it from a central network location.

**Comment by eirannejad** [comment]
Look at `pyrevit clone --help`. It has 3 different variations

```
pyrevit clone <clone_name> <deployment_name> [--dest=<dest_path>] [--branch=<branch_name>] [--log=<log_file>]
pyrevit clone <clone_name> --image=<image_url> [--dest=<dest_path>] [--log=<log_file>]
pyrevit clone <clone_name> [--dest=<dest_path>] [--source=<repo_url>] [--branch=<branch_name>] [--log=<log_file>] [--username=<username>] [--password=<password>]
```

You need to use this to deploy from an **image**
`pyrevit clone <clone_name> --image=<image_url> --dest=<dest_path> --log=<log_file>`

There is no "<deployment_name>" since it is assumed the image is custom and is already setup they way it needs to be

**Comment by fabioloretti** [comment]
Thanks for the assistance, Ehsan. My goal was to simply create a local clone of your Core deployment but use a local repository to create all the user's clones as opposed to having all users download the data.
I tried creating a clone from github using the line below, which works.
pyrevit clone cloneName core --dest=C:\Path

I then zipped the folder C:\Path\cloneName to use as a central "custom" deployment, which I could place in the server and call from each user's machine during installation using:
pyrevit clone cloneName --source=\\network\cloneName.zip --dest=C:\Path

Unfortunately the above didn't work. Any help would be very much appreciated. Thank you.
Fabio
